doc/z2csl.svg

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Yeah, I think I agree here. I thought it might look more useful and therefore tried it, but I admit that it might only be a slightly different perspective than looking at the XML file itself. I will delete the diagram and XSD file.